### PR TITLE
[Model] Add the Infinity-Instruct SFT code

### PR DESCRIPTION
**Pull request by CathySama**

### CODE REVIEW

File: examples/qwen/conf_qwen/train/train_qwen_2.5_1.5b.yaml
Comment (aoyulong) [inline examples/qwen/conf_qwen/train/train_qwen_2.5_1.5b.yaml:1]: Remove the `train_` prefix from the file names since they are in the `train` directory.

File: examples/qwen/dist_start.sh
Comment (aoyulong) [inline examples/qwen/dist_start.sh:1]: We have updated the launcher to use the unified `run.py`. Please remove `dist_start.sh`, `dist_stop.sh`, `env.sh`, and `run.sh`.

File: flagscale/train/train_aquila.py [hunk 1]
```diff
@@ -165,12 +165,13 @@ def loss_func(loss_mask: torch.Tensor, output_tensor: torch.Tensor):
 
     losses = output_tensor.float()
     loss_mask = loss_mask.view(-1).float()
+
     total_tokens = loss_mask.sum()
-    loss = torch.cat([torch.sum(losses.view(-1) * loss_mask).view(1), total_tokens.view(1)])
+
+    loss = torch.cat([torch.sum(torch.masked_select(losses.view(-1) , loss_mask==1)).view(1), total_tokens.view(1)])
```
Comment (aoyulong) [inline flagscale/train/train_aquila.py]: This loss will also be used in pre-training. If the SFT requires a different one, we may need a better way to distinguish them.

File: examples/qwen/dist_start.sh
Comment (CathySama) [reply examples/qwen/dist_start.sh:1]: These files (dist_start.sh, dist_stop.sh, env.sh, run.sh and the directories of tokenizers)  have been removed.

File: flagscale/train/train_aquila.py (hunk 1 above)
Comment (CathySama) [reply flagscale/train/train_aquila.py]: I add a new file called train_aquila_sft.py to distinguish the loss.

**Review by aoyulong — APPROVED** [review]
LGTM

**Comment by CathySama** [comment]
Have renamed conf_qwen.
